src/test/functional/register-user-test.ts

```diff
@@ -9,28 +9,30 @@ import {config as testConfig} from '../config';
 import * as Assert from 'assert';
 import {randomData} from './shared/random-data';
 import {BETA_ADD, GAMMA_PRIVATE_BETA} from '../../main/app/feature-flags/flags';
-import {UserType} from '../../main/utils/UserType';
 
 Feature('Register New User');
 
 const PARENT_ROLE = randomData.getRandomRole();
 const ASSIGNABLE_CHILD_ROLE1 = randomData.getRandomRole();
 const ASSIGNABLE_CHILD_ROLE2 = randomData.getRandomRole();
+const INDEPENDENT_ROLE = randomData.getRandomRole();
 const DASHBOARD_USER_EMAIL = randomData.getRandomEmailAddress();
 const SERVICE_WITH_PRIVATE_BETA = randomData.getRandomRole();
-const PRIVATE_BETA_USER_ROLE = SERVICE_WITH_PRIVATE_BETA + '-beta-role';
+const RANDOM_SERVICE = randomData.getRandomRole();
 
 const OAUTH_REDIRECT_URI = 'http://test.com/oauth2/callback';
 
 BeforeSuite(async () => {
   await createAssignableRoles(PARENT_ROLE);
   await createAssignableRoles(ASSIGNABLE_CHILD_ROLE1);
   await createAssignableRoles(ASSIGNABLE_CHILD_ROLE2);
-  await createAssignableRoles(PRIVATE_BETA_USER_ROLE);
+  await createAssignableRoles(SERVICE_WITH_PRIVATE_BETA);
+  await createAssignableRoles(INDEPENDENT_ROLE);
   // Assigning self role with the child role so the this user can also delete same level users
-  await assignRolesToParentRole(PARENT_ROLE, [ASSIGNABLE_CHILD_ROLE1, ASSIGNABLE_CHILD_ROLE2, PARENT_ROLE, PRIVATE_BETA_USER_ROLE, UserType.Citizen]);
+  await assignRolesToParentRole(PARENT_ROLE, [ASSIGNABLE_CHILD_ROLE1, ASSIGNABLE_CHILD_ROLE2, PARENT_ROLE, SERVICE_WITH_PRIVATE_BETA, testConfig.USER_ROLE_CITIZEN]);
   await createUserWithRoles(DASHBOARD_USER_EMAIL, testConfig.PASSWORD, testConfig.USER_FIRSTNAME, [testConfig.RBAC.access, PARENT_ROLE]);
-  await createService(SERVICE_WITH_PRIVATE_BETA, SERVICE_WITH_PRIVATE_BETA, SERVICE_WITH_PRIVATE_BETA, SERVICE_WITH_PRIVATE_BETA, [OAUTH_REDIRECT_URI], [PRIVATE_BETA_USER_ROLE]);
+  await createService(SERVICE_WITH_PRIVATE_BETA, SERVICE_WITH_PRIVATE_BETA, SERVICE_WITH_PRIVATE_BETA, SERVICE_WITH_PRIVATE_BETA, [OAUTH_REDIRECT_URI], [SERVICE_WITH_PRIVATE_BETA]);
```

Comment:
Not sure why u want to replace PRIVATE_BETA_USER_ROLE with SERVICE_WITH_PRIVATE_BETA here. This is an onboarding role and not a service so it is rather misleading

Reply:
It was same variable you were passing in this, just different name

Reply:
Gave seperate name to role